src/main.rs

```diff
@@ -36,6 +37,7 @@ fn main() {
         ],
     );
 
-    let server = SpaceapiServer::new(host, status);
+    let datastore = datastore::RedisStore::new().unwrap();
+    let server = SpaceapiServer::new(host, status, datastore);
```

Comment:
See #32: We need Mutex for Sync, Arc for Send and Box to hold a trait object. Resulting in the ```Arc<Mutex<Box<T>>>``` composition